src/lightning/fabric/accelerators/tpu.py

```diff
@@ -91,45 +102,26 @@ def register_accelerators(cls, accelerator_registry: Dict) -> None:
 TPU_CHECK_TIMEOUT = 60
 
 
-def _inner_f(queue: Queue, func: Callable, *args: Any, **kwargs: Any) -> None:  # pragma: no cover
-    try:
-        queue.put(func(*args, **kwargs))
-    except Exception:
-        traceback.print_exc()
-        queue.put(None)
-
-
-def _multi_process(func: Callable) -> Callable:
-    @functools.wraps(func)
-    def wrapper(*args: Any, **kwargs: Any) -> Union[bool, Any]:
-        queue: Queue = Queue()
-        proc = Process(target=_inner_f, args=(queue, func, *args), kwargs=kwargs)
-        proc.start()
-        proc.join(TPU_CHECK_TIMEOUT)
-        try:
-            return queue.get_nowait()
-        except q.Empty:
-            traceback.print_exc()
-            return False
-
-    return wrapper
+def _inner_f(queue: Queue, func: Callable) -> None:
+    res = func()
+    queue.put(res)
 
 
-@_multi_process
-def _is_device_tpu() -> bool:
-    """Check if TPU devices are available. Runs XLA device check within a separate process.
+def _has_tpu_device() -> bool:
+    """Check if TPU devices are available.
 
     Return:
         A boolean value indicating if TPU devices are available
     """
     if not _XLA_AVAILABLE:
         return False
     import torch_xla.core.xla_model as xm
+    from torch_xla.experimental import pjrt
 
-    # For the TPU Pod training process, for example, if we have
-    # TPU v3-32 with 4 VMs, the world size would be 4 and as
-    # we would have to use `torch_xla.distributed.xla_dist` for
-    # multiple VMs and TPU_CONFIG won't be available, running
+    if pjrt.using_pjrt:
```

Comment:
It need to be called as a func: `pjrt.using_pjrt()`

Reply:
Ah good eye, silly mistake

```suggestion
    if pjrt.using_pjrt():
```